plugins/view-resources/src/components/navigator/TreeElement.svelte

```diff
@@ -47,13 +48,25 @@
   export let actions: (originalEvent?: MouseEvent) => Promise<Action[]> = async () => []
 
   let hovered = false
+  let inlineActions: Action[] = []
+  let popupMenuActions: Action[] = []
+
+  $: actions().then((result) => {
+    inlineActions = result.filter((action) => action.inline === true)
+    popupMenuActions = result.filter((action) => !action.inline === true)
```

Comment:
`!action.inline === true` -> `action.inline === false` or `action.inline !== true` if it is nullable